torchbenchmark/models/DALLE2_pytorch/LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
```

Comment:
We don't need to keep the `DALLE2_pytorch` directory, we only need to specify the commit hash in `requirements.txt` and in `install.py`, specify `pip install requirements.txt` to install a specific commit of this model.